iroh/src/commands/list.rs

```diff
@@ -13,8 +15,8 @@ pub enum Commands {
     Blobs,
     /// List the available blobs on the running provider.
     IncompleteBlobs,
-    /// List the available collections on the running provider.
-    Collections,
+    /// List the available roots on the running provider.
+    Roots,
```

Comment:
can roots be both `blobs` & `collections`?

Reply:
ahhh i see from other parts of the code that yes they can be. 

I understand why you mentioned that naming has been challenging for this PR. I'll have to mull this over.

Reply:
Yes, if we end up doing the ipfs format flag you could even have any multiformat here.